lib/rspec/mocks/proxy.rb

```diff
@@ -338,9 +341,9 @@ def reset
         super
       end
 
-      def message_received(message, *args, &block)
+      def message_received(message, *args, **opts, &block)
         RSpec::Mocks.space.any_instance_recorders_from_ancestry_of(object).each do |subscriber|
-          subscriber.notify_received_message(object, message, args, block)
```

Comment:
Wondering if we can pass block as `&block` and get rid of `_block` parameter in the `notify_received_message` method signature.